docs/getting-started/what-is-dotnet.md

```diff
@@ -95,15 +95,21 @@ Other libraries are installed as [NuGet](http://www.nuget.org/ "NuGet") packages
+.NET Core includes a new cross-platform command-line SDK known as the CLI (for Command-Line Interface). The CLI is a UNIX-friendly set of tools for building .NET Core applications. It abstracts the C# compiler, the NuGet package manager, and includes incremental build logic. It can also integrate with the .NET Native tool chain to produce high-performance native apps and libraries. The CLI allows developers to build and test applications with a minimal installation on developer, lab, or production machines. Since some IDEs, such as Visual Studio Code and Visual Studio, use the CLI under the covers, developers can now choose the level they want to interact with the tool chain by working with the CLI directly, using an editor that uses the CLI, or developing with an IDE that uses the CLI internally.  
 
 ## Language Support and Development Environments ##
 
-.NET Core provides a runtime environment for your app, as well as a collection of libraries whose types and members your application can use. .NET Core itself is language agnostic: it is not a programming language, but any programming languages that targets .NET Core can be used to develop .NET Core applications, and an application written in one programming language can seamlessly access types and members in libraries developed by using a different programming language. 
+.NET Core provides a runtime environment for your app, as well as a collection of libraries whose types and members your application can use. .NET Core itself is language agnostic: it is not a programming language, but any programming language that targets .NET Core can be used to develop .NET Core applications, and an application written in one programming language can seamlessly access types and members in libraries developed by using a different programming language. 
  
-You develop and compile .NET Core apps by using the [.NET Compiler Platform](https://github.com/dotnet/roslyn "Roslyn Compiler Platform"), also known as Roslyn, which allows you to write your apps in C# or F#. (At the moment, other languages are not supported, though that will change going forward.) 
+You develop and compile .NET Core apps in either of two languages:
 
-You have a wide choice of development environments for writing your app, ranging from a simple text editor or programmer's editor of your choice to [Visual Studio Code](https://code.visualstudio.com/ "Visual Studio Code"), to [Visual Studio 2015](https://www.visualstudio.com/en-us/visual-studio-homepage-vs.aspx "Visual Studio 2015"), to [Unity](http://unity3d.com/unity "Unity").   
+- C#, which is based on the [.NET Compiler Platform](https://github.com/dotnet/roslyn "Roslyn Compiler Platform"), also known as Roslyn. 
+
+- [F#](https://channel9.msdn.com/Events/Build/2016/T661 "F#").
+ 
+(At the moment, other languages are not supported, though that will change going forward.) 
```

Comment:
Ask for PM review, we can never promise. I'd rewrite it as

We [plan | hope] to support VM and other languages in the future.

going forward -> marketing